scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -462,6 +463,7 @@ task M2 {
     Int? preemptible_tries
   }
 
+  Int max_reads_per_alignment_start_arg = select_first([max_reads_per_alignment_start, 75])
```

Comment:
Is there a reason why you don't initialize this in the `input` section instead of making this optional? e.g.,
```
input {
   ...
   Int max_reads_per_alignment_start = 75
   ...
}
```
This lets a caller override it but since it's not optional you don't need to `select_first()` on it.

Reply:
When I tried that it didn't show up in Terra. That would definitely be preferable.

Reply:
Well I have no experience with Terra inputs, so maybe there's no better way to do it then. I know it's not visible until `allowNestedInputs: true` is set.. but once you do that, you should be able to pass through any workflow inputs.

Reply:
megan i think i got this to work. see https://portal.firecloud.org/?return=terra#methods/andrea_methods/MitochondriaPipeline_experiment/4 (which uses an AlignAndCallTest workflow that defaults this arg to 75).

Reply:
nevermind - was looking at the wrong version